scipy/interpolate/predict.py

```diff
@@ -0,0 +1,114 @@
+""" This module implements the Always Stable Projector from Ref. [1] that can
+be used for predicting values in a (time) series, such as in [2-3].
+
+[1] J. Kolafa, J. Comput. Chem. 25, 335 (2004).
```

Comment:
Can you add this reference to the docstring of the function/class (whichever one will be kept) instead? See https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt for desired formatting.